server/routers/api-router.js

```diff
@@ -214,6 +216,193 @@ router.get("/api/status-page/heartbeat", cache("5 minutes"), async (_request, re
     }
 });
 
+router.get("/api/badge/:id/status", cache("5 minutes"), async (request, response) => {
+    allowAllOrigin(response);
+
+    const {
+        label,
+        upLabel = "Up",
+        downLabel = "Down",
+        upColor = badgeConstants.defaultUpColor,
+        downColor = badgeConstants.defaultDownColor,
+        style = "flat",
```

Comment:
`style = "flat",` is repeated 3 times in this file

Reply:
Well, there are several code blocks that share similarities / repeat themselves. 
Would you like to see a constant here as well or a more sophisticated routing approach in which 'ping' and  'uptime' share a route?